backup/setup/systems.json

```
@@ -18,7 +18,8 @@
 		"_id": {
 			"$oid": "0000d186816abba584714c92"
 		},
-		"type": "iserv",
+		"alias": "Zentrales IServ",
```

Please change it to "OAuth Zentrales IServ" an "Zentrales IServ" will be add for the developer Instances via [init script](https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/ansible/roles/schulcloud-server-init/templates/configmap_file_init_db.yml.j2#L30).
```suggestion
		"alias": "OAuth Zentrales IServ",
```